ci/run_example_tests.sh

```diff
@@ -0,0 +1,470 @@
+    name=mysql
+    paths=mysql
+    mysql_client=(docker run -ti --network envoymesh mysql:5.5 mysql -h envoy -P 1999 -u root)
+
+    bring_up_example "$name" "$paths" 10
+
+    run_log "$name" "Create a mysql database"
+    "${mysql_client[@]}" -e "CREATE DATABASE test;"
+    "${mysql_client[@]}" -e "show databases;"
+
+    run_log "$name" "Create a mysql table"
+    "${mysql_client[@]}" -e "USE test; CREATE TABLE test ( text VARCHAR(255) );"
+    "${mysql_client[@]}" -e "SELECT COUNT(*) from test.test;"
+
+    run_log "$name" "Check mysql egress stats"
+    curl -s http://localhost:8001/stats?filter=egress_mysql
+
+    run_log "$name" "Check mysql TCP stats"
+    curl -s http://localhost:8001/stats?filter=mysql_tcp
+
+    cleanup "$name" "$paths"
+}
+
+run_example_zipkin_tracing () {
+    local name paths
+    name=zipkin
+    paths=zipkin-tracing
+    bring_up_example "$name" "$paths"
+
+    run_log "$name" "Test connection"
+    curl -v localhost:8000/trace/1  2> >(grep -v Expire)
+
+    run_log "$name" "Test dashboard"
+    # this could do with using a healthcheck and waiting
+    sleep 20
+    curl localhost:9411/zipkin/
+
+    cleanup "$name" "$paths"
+}
+
+run_example_front_proxy () {
+    local name paths
+    name=front_proxy
+    paths=front-proxy
+    bring_up_example "$name" "$paths"
+
+    run_log "$name" "Test service: localhost:8080/service/1"
+    curl -v localhost:8080/service/1 2> >(grep -v Expire)
+    run_log "$name" "Test service: localhost:8080/service/2"
+    curl -v localhost:8080/service/2 2> >(grep -v Expire)
+    run_log "$name" "Test service: https://localhost:8443/service/1 -k -v"
+    curl https://localhost:8443/service/1 -k -v 2> >(grep -v Expire)
+    run_log "$name" "Scale up docker service1=3"
+    docker-compose scale service1=3
+
+    run_log "$name" "Snooze for 5 while docker-compose scales..."
+    sleep 5
+
+    curl -v localhost:8080/service/1 2> >(grep -v Expire)
+    run_log "$name" "Test round-robin localhost:8080/service/1"
+    docker-compose exec front-envoy bash -c "\
+    		   curl localhost:8080/service/1 \
+		   && curl localhost:8080/service/1 \
+		   && curl localhost:8080/service/1"
+    run_log "$name" "Test service: localhost:8080/service/2"
+    docker-compose exec front-envoy curl localhost:8080/service/2 2> >(grep -v Expire)
+    run_log "$name" "Test service info: localhost:8080/server_info"
+    docker-compose exec front-envoy curl localhost:8001/server_info | jq '.'
+    run_log "$name" "Test service stats: localhost:8080/stats"
+    docker-compose exec front-envoy curl localhost:8001/stats
+
+    cleanup "$name" "$paths"
+}
+
+
+show_user_env
+clone_envoy_repo
+wait_for_docker
+run_example_cors
+run_example_csrf
+run_example_ext_authz
+run_example_fault_injection
+run_example_front_proxy
+run_example_grpc_bridge
+run_example_jaeger_native_tracing
+run_example_jaeger_tracing
+run_example_load_reporting
+run_example_lua
+run_example_mysql
+run_example_zipkin_tracing
```

Comment:
Is there a way to run examples without listing examples here?

Reply:
yep, i can update